pyspark/bigdl/keras1/converter.py

```diff
@@ -403,16 +408,218 @@ def create_reshape(self, klayer, kclayer):
         blayer = BLayer.Reshape(klayer.target_shape, None)
         return blayer
 
+    def create_repeatvector(self, klayer, kclayer):
+        return BLayer.Replicate(n_features=klayer.n,
+                                n_dim=1,
+                                bigdl_type="float")
+
     def create_merge(self, klayer, kclayer):
         self.__check_is_share_weights(kclayer)
+        input_shape = klayer.get_input_shape_at(0)
```

Comment:
add exception if those parameters are not None: `output_shape=None, output_mask=None,
                 arguments=None, node_indices=None, tensor_indices=None`